src/families/polkadot/logic.js

```diff
@@ -239,3 +239,11 @@ export const calculateAmount = ({
 
   return amount.lt(0) ? BigNumber(0) : amount;
 };
+
+export const getExistentialDeposit = (a: Account): BigNumber => {
```

Comment:
I find this a bit confusing with actual `EXISTENTIAL_DEPOSIT` of 1 DOT, could rename that function `getMinimumBalance` maybe?